packages/react-ui-components/src/stories/Components/Search.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { ComponentStory } from '@storybook/react';
+import React from 'react';
+import { EComponentSize, KvSearch, KvTextField } from '../../components';
+
+KvTextField.displayName = 'KvSearch';
+
+export default {
+	title: 'Inputs/Search',
+	component: 'kv-search',
+	argTypes: {
```

Comment:
It's possible to add here the events that are generated by the component?
I think it's cool to have this. 

[Look Here](https://storybook.js.org/docs/react/essentials/actions)

Reply:
It is possible! But every component is lacking this feature. 
@ricardogoncalves89